docs/STAFF.md

```
@@ -0,0 +1,937 @@
+    float          scroll_accum;      // +1380 剩余像素距离（到显示下一行）
+    StaffScrollCmd *cmd_ptr;          // +1384 当前命令表读取指针
+};
+```
+
+---
+
+### 渲染管线（逐帧流程）
+
+#### 第一阶段：初始化 `StaffRoll_Init` @ `0x8975BB4`
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)